COLLABORATOR_GUIDE.md

```diff
@@ -73,6 +73,77 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+## Breaking Changes
+
+Backwards-incompatible changes may land on the master branch at any time after
+sufficient review by collaborators and approval of at least two CTC members.
+
+Examples of breaking changes include, but are not necessarily limited to,
+removal or redefinition of existing API arguments, changing return values,
+adding or removing errors, altering the timing, and changing the non-internal
```

Comment:
In general it *should* be, I think, but I've come across cases (as awful as it is) where thrown errors are used for flow control. Removing an expected error, then, can lead to code no longer functioning correctly. For instance, `glob` currently appears to rely on [`ENAMETOOLONG`](https://github.com/isaacs/node-glob/blob/master/glob.js#L597) errors to know when to bail out to avoid recursing endlessly in certain situations. Given how sensitive error handling is, it's often best to err on the side of caution than to make assumptions about what things ought to be.